src/PageNav.tsx

```diff
@@ -81,12 +81,12 @@ export const PageNav: React.FunctionComponent = () => {
             </NavItem>
           </NavList>
           {!isOnAddRealm && (
-            <NavGroup title="">
+            <NavGroup aria-labelledby="" title="">
               <LeftNav title="home" path="/" />
             </NavGroup>
           )}
           {showManage && !isOnAddRealm && (
-            <NavGroup title={t("manage")}>
+            <NavGroup aria-labelledby={t("manage")} title={t("manage")}>
```

Comment:
Be aware that `aria-labelledby` references the `id` of an element that contains a descriptive text and should not be confused with `aria-label` which provides a text directly.

So probably what we need to do here is:
```suggestion
            <NavGroup aria-label={t("manage")} title={t("manage")}>
```

Reply:
Good suggestion! thank you for clarifying